forwards_paper.tex

```diff
@@ -771,15 +859,23 @@ \subsection*{Tree sequence simplification}
 % tree sequence representing a large dataset and wish to extract the
 % information relevant to a subset of the samples.
 
-Our approach to simplification is based on Hudson's algorithm for simulating
-the coalescent with recombination~\citep{hudson1983properties},
-paralleling the implementation in \msprime{} \citep{kelleher2016efficient};
-an implementation in pseudocode is provided in Appendix~\ref{ss:simplify_algorithm},
+We implement simplification by starting at the end of the simulation,
+and moving back up through history,
+recording in the new tree sequence only that information necessary to construct the tree sequence
+of the specified individuals.
+This process of tracing ancestry back through time in a pedigree
+was the motivation for Hudson's coalescent simulation algorithm~\citep{hudson1983properties},
+so it is unsurprising that simplification uses many of the same tools
+as the implementation of Hudson's algorithm in \msprime{} \citep{kelleher2016efficient}.
+The main difference is that events in a coalescent simulation are random, \revpoint{1}{2}
+while here they are predetermined by history.
```

Comment:
s/are random/occur at random times/

and

s/here they are/here these timings are/

Reply:
I'd say that there's more to the randomness than just the times: for instance, if you did a coalescent simulation in a known pedigree, you'd stlil generate random breakpoints.

Reply:
so I want to keep the first "are random"

Reply:
Sounds good.

The key thing is that the coalescent simulation *algorithm* includes randomness whereas our simplication algorithm, is deterministic given a history. That's probably already explicit enough and I was just being dense. If not

s/here/in our simplication algorithm/

makes it more explicit